charts/ziti-controller/templates/configmap.yaml

```diff
@@ -336,7 +336,7 @@ data:
             # address - required
             # The public address that external incoming requests will be able to resolve. Used in request processing and
             # response content that requires full host:port/path addresses.
-            address: {{ .Values.prometheus.advertisedHost | default (printf "%s-prometheus.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}:{{ .Values.prometheus.advertisedPort }}
```

Comment:
This section configures the optional web listener binding for the Prometheus scrape target when `prometheus.service.enabled` is true. The conventional cluster service suffix is `-prometheus`. `-ctrl` is the suffix for the ctrl plane, which is a different cluster service. The domain names with suffixes are important because that's how the ingress controller distinguishes requests to route them appropriately.

Reply:
bad copy pasta here too!